packages/strapi-generate-new/lib/resources/dot-files/gitignore

```diff
@@ -110,5 +109,7 @@ coverage
 .env
 license.txt
 exports
-.cache
+*.cache
 build
+documentation
```

Comment:
This needs to stay out of this git ignore. It is in the example to avoid getting PRs with documentation updates but in normal usage it needs to be committed.